src/coredata.cs

```diff
@@ -305,6 +381,12 @@ public interface NSFetchRequestExpression {
 		bool IsCountOnly { [Bind ("isCountOnlyRequest")] get;}
 	}
 
+	[Watch (3,0), TV (10,0), iOS (10,0), Mac (10,12)]
```

Comment:
in this case the tribute are not needed. The `I*` type is just to get the definition to compile (as correct C#) and it's not used by the generator itself.